src/Components/Home.js

```diff
@@ -63,85 +69,68 @@ function HomePage() {
           </button>
         </div>
 
-        <img
-          className={`uw-logo ${darkMode ? "dark-mode" : ""}`}
-          src={darkMode ? UWlogo2 : UWlogo}
-          alt="UW logo"
-        />
-        <h1 className="clubName" id={"clubName"}>
-          Software Engineering Career Club
-        </h1>
+        <div className="club-introduction-container">
+          <h1>Software Engineering Career Club</h1>
+          <p>at the University of Washington</p>
+          <button
+            className="club-introduction-container-button"
+            onClick={() => scrollToSection("whyJoinUs")}
+          >
+            Learn More
+          </button>
+        </div>
       </div>
     );
   }
 
-  const meetingInfo = (
-    <div className="meetingsSection">
-      <div className="meetingsInfoSection">
-        <h2 className="sweccMeetings">SWECC Meetings</h2>
-        <p className="meetingInfo">
-          Held in the Loew 216, these meeting feature topics in a vast number of
-          areas, including professional development, resume building, and mentor
-          circles
-        </p>
-        <h3 className="meetingTime">Meeting Times/Location (Spring 2024)</h3>
-        <p className="meetingDetails">
-          Every week on Wednesday, from 5:30-6:30pm
-          <br />
-          Location: Loew 216
-        </p>
-      </div>
-      <div className="map-and-label">
-        <img className="map" src={SWECCmap} alt="map" />
-        <p className="mapLabel">Location: Loew 216</p>
-      </div>
-    </div>
-  );
-
-  const posts = (
-    <div className="posts">
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/C2-o7roPbfN"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/C23PyAWPYdW"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/CyRondZPVVS"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/Cxum20yOlkQ"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/CxrxWcQLpSb"
-      />
-    </div>
-  );
+  function intPostsWidget() {
+    return <BeholdWidget feedId="5rAX7PhyjFjmyVfW4Plm" />;
```

Comment:
InstagramPostsWidget?

also should prob change to

```js
function InstagramPostsWidget() {
  return <BeholdWidget ... />;
}
```

so you can later use it like 

```js
<InstagramPostWidget/>
```